app/src/views/Header.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+const Header = () => {
```

Comment:
I would add this in the components file since it is not a view in itself. The header title will also be confusing since all our views have "headers". I like the separation of concerns though and keeping it a dummy function component.  I'd reanme it to ToolHeader and move to components and its should be good.